app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -36,4 +37,9 @@ class AdministratorControlsActivity : InjectableAppCompatActivity() {
       return KEY_NAVIGATION_PROFILE_ID
     }
   }
+
+  override fun routeToProfileList() {
+    val intent = Intent(this, ProfileListActivity::class.java)
```

Comment:
Instead of creating an intent here. We should create a companion object inside `ProfileListActivity` which will be called from here. Checkout the implementation of such intent transfers in various other activities.

Reply:
Done. I thought about this but as it was not created before, I thought it should be kept as it is.